addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/rest/graph/ProjectTraversalResourceImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.jboss.windup.web.addons.websupport.rest.graph;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.model.OrganizationModel;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.web.addons.websupport.model.PersistedProjectModelTraversalModel;
+import org.jboss.windup.web.addons.websupport.model.PersistedTraversalChildFileModel;
+import org.jboss.windup.web.addons.websupport.services.PersistedProjectModelTraversalService;
+
+/**
+ * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ */
+public class ProjectTraversalResourceImpl extends AbstractGraphResource implements ProjectTraversalResource
```

Comment:
Could you please enlight this a bit with a Javadoc... everything around traversals needs extra javadoc :)